source/support.tex

```diff
@@ -571,17 +574,20 @@
 #define @\defnlibxname{cpp_lib_byte}@                              201603L // also in \libheader{cstddef}
 #define @\defnlibxname{cpp_lib_char8_t}@                           201907L
   // also in \libheader{atomic}, \libheader{filesystem}, \libheader{istream}, \libheader{limits}, \libheader{locale}, \libheader{ostream}, \libheader{string}, \libheader{string_view}
-#define @\defnlibxname{cpp_lib_chrono}@                            201907L // also in \libheader{chrono}
+#define @\defnlibxname{cpp_lib_chrono}@                            201803L  // also in \libheader{chrono}
```

Comment:
Does P1902R1 really mean to bump *down* the value of `__cpp_lib_chrono`?

Reply:
@BRevzin

Reply:
@burblebee Gah! This is my mistake. R0 of the paper introduced a new macro for the date library and the suggestion was to bump the preexisting chrono macro instead. 

When I changed it to bump the old macro, I didn't look at what the current value was.

The intent was an intermediate bump, so no change in value overall 

My mistake and good catch!

Reply:
Ok, reverted back to 201907L.  @brevzin If this was the wrong thing to do, please unresolve this.  Thanks!

Reply:
@burblebee Looks good to me!